### PR TITLE
Feature/sample data

### PR DESCRIPTION
**Pull request by mingxin-yu**
Create a room data structure
Create a furniture data structure
Create a sample database
Adding support file here and there

### CODE REVIEW

File: src/app/room.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export interface Room {
+  roomId: number;
```
Comment (ghost) [inline src/app/room.ts]: if the id is of the entity, just name it as id.

File: src/app/furniture.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export interface Furniture {
+  furnitureId: number;
```
Comment (ghost) [inline src/app/furniture.ts]: if the id is of the entity, just name it as id.

File: src/app/room.ts (hunk 1 above)
Comment (mingxin-yu) [reply src/app/room.ts]: Changed

File: src/app/furniture.ts (hunk 2 above)
Comment (mingxin-yu) [reply src/app/furniture.ts]: Changed

**Comment by ghost** [comment]
LGTM

**Pull request merged**